examples/ui-demo/src/components/preview/AuthCardWrapper.tsx

```diff
@@ -17,18 +23,20 @@ export function AuthCardWrapper({ className }: { className?: string }) {
         className
       )}
     >
-      <RenderContent hasUser={!!user} hasClient={!!client} />
+      <RenderContent user={user} hasClient={!!client} />
     </div>
   );
 }
 
 const RenderContent = ({
-  hasUser,
+  user,
   hasClient,
 }: {
-  hasUser: boolean;
+  user: UseUserResult;
```

Comment:
seems odd to add drill this down, we should just use the hook directly here since it pulls user state from a static context

Reply:
I think moldy's point here was to not send down a prop here and at line 38 just use the `useUser` hook. Any reason we can't do that @iykazrji ?

Reply:
Yeah @rthomare, it's been updated now to do exactly that. Thanks for taking a look @rthomare @moldy530 !